vm/src/obj/objobject.rs

```diff
@@ -190,6 +195,19 @@ fn object_init(vm: &mut VirtualMachine, _args: PyFuncArgs) -> PyResult {
     Ok(vm.ctx.none())
 }
 
+fn object_class(_obj: PyObjectRef, owner: PyObjectRef, _vm: &mut VirtualMachine) -> PyObjectRef {
```

Comment:
Using a new style function here loses the type check on owner. (We need to implement PyClassRef to support that.)

Reply:
Rather than having to switch back to old style, how about a "TODO: owner should be PyClassRef once that exists"?